src/components/Notification.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+interface NotificationProps {
+  error: string,
+}
+
+export const Notification = ({ error }:NotificationProps) => {
+  const [isHidden, setIsHidden] = useState(false);
+
+  return (
+    <div className={`notification is-danger is-light has-text-weight-normal ${isHidden ? 'hidden' : ''}`}>
+      <button
+        type="button"
+        className="delete"
+        onClick={() => setIsHidden(true)}
+      />
+      {error}
+    </div>
```

Comment:
May looks better with classnames.
```suggestion
<div className={cn(
  'notification',
  'is-danger',
  'is-light',
  'has-text-weight-normal',
  { hidden: isHidden }
)}>
  <button
    type="button"
    className="delete"
    onClick={() => setIsHidden(true)}
  />
  {error}
</div>
```